salt/modules/scsi.py

```diff
@@ -14,7 +15,7 @@
 }
 
 
-def ls_():
+def ls_(get_size=True):
     '''
     List SCSI devices, with details
 
```

Comment:
Can you add a definition for `get_size` here, as well as a `.. versionadded:: 2015.5.10` directive for that new kwarg?